CardiorespiratoryFitness/CardiorespiratoryFitness/Shared Resources/Images.xcassets/Activity - Heart Snapshot/heartSnapshotIcon.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "filename" : "ic_heart_snapshot 1.png",
```

Comment:
Use PDF image.

Reply:
Curious on your thoughts here?  Both png and pdf are valid for the image catalog

Reply:
Also, not sure if you are aware, but exciting news xCode 12+ added support for SVG

Reply:
Nice. sag would be fine as well.